dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-hive/src/main/java/org/apache/dolphinscheduler/plugin/datasource/hive/HiveDataSourceClient.java

```diff
@@ -90,7 +91,8 @@ protected void checkEnv(BaseConnectionParam baseConnectionParam) {
 
     private void checkKerberosEnv() {
         String krb5File = PropertyUtils.getString(JAVA_SECURITY_KRB5_CONF_PATH);
-        if (StringUtils.isNotBlank(krb5File)) {
```

Comment:
Whether krb5file is empty or not may not be removed

Reply:
Did you mean kerberosStartupState and krb5file double check?

Reply:
This is the best. Krb5file does not exist, and problems are likely to occur

Reply:
I'll fix it.thanks.